CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+### Changed
+- Updated buildkit to include changes up to [a5263dd0f990a3fe17b67e0002b76bfd1f5b433d](https://github.com/moby/buildkit/commit/a5263dd0f990a3fe17b67e0002b76bfd1f5b433d)
```

Comment:
Did you want to add a note about how this includes an improvement to speed up buildkit start time?

Reply:
maybe we'll leave it as an easter-egg; I don't think any users ever reported the slowness -- it's just something we discovered.